toolkit/imageconfigs/marketplace-gen2-aarch64.json

```diff
@@ -75,7 +75,12 @@
             "KernelCommandLine": {
                 "ExtraCommandLine": "console=tty1 console=ttyAMA0 earlycon=pl011,0xeffec000 initcall_blacklist=arm_pmu_acpi_init"
             },
-            "Hostname": "cbl-mariner"
+            "Hostname": "cbl-mariner",
+            "FinalizeImageScripts": [
```

Comment:
@dmcilvaney I am using a FinalizeImageScript to call mkconfig from within the image. 

This is because during InstallGrubCfg we are still in the installroot, which seems to be a different environment than the image.